SECURITY.md

```diff
@@ -0,0 +1,29 @@
+# Security
+
+GitHub takes the security of our software products and services seriously, including all of the open source code repositories managed through our GitHub organizations, such as [GitHub](https://github.com/GitHub).
```

Comment:
Replace references to Github with Infraspec

Reply:
Done 63f15249abf6fbbd914c2d5932e619df45e1c715

Reply:
I still see Github mentioned, you can use a find replace on all references to Github. 
"GitHub takes the security of.." should be "Infraspec takes the security of.." as this doc is saying we (Infraspec) take security seriously.
for reference, [here](https://github.com/microsoft/.github/blob/main/SECURITY.md)

Reply:
Done 8431e0c1c455b8e0c00535952e711fa460562476